lodash.src.js

```diff
@@ -3707,6 +3707,25 @@
     }
 
     /**
+     * Creates a `_.round`, `_.ceil`, or `_.floor` function.
+     *
+     * @private
+     * @param {String} type Specify the Math method to use when rounding.
+     */
+
+    function createRound(type) {
+      var roundFn = Math[type];
+      return function(number, precision) {
+        if (precision) {
+          precision = Math.pow(10, precision);
```

Comment:
You [know how I do](https://github.com/lodash/lodash/commit/c8b818f011257b09f98d12fabde92ba5a77a43d3#diff-a0ac9c4b57e4a35674426f2b910a39daR754).